shell/probe/nocturnal.sh

```diff
@@ -18,6 +18,12 @@ do
     else
         rm -rf $vst
         unset dat
-        sleep 3600
+
+        # Option 1. Explicitly check that sleep is an int. Otherwise default to 1800s
```

Comment:
I am a fan of the explicity you have in option 1. I would approve that option.

Reply:
kewl. patrick agrees

out of curiosity, can you think of a way to take advantage of option 2?

Reply:
I honestly can not think of anything worth the effort.